service/src/main/scala/org/apache/celeborn/server/common/http/HttpEndpoint.scala

```diff
@@ -39,6 +42,22 @@ case object Conf extends HttpEndpoint {
     service.getConf
 }
 
+case object DynamicConfig extends HttpEndpoint {
+  override def path: String = "/dynamicConfig"
+
+  override def description(service: String): String = s"List the dynamic configs of the $service. " +
+    s"The parameter level specifies the config levels of dynamic configs, which value is separated by commas. " +
+    s"The parameter tenant specifies the tenant ids of ${ConfigLevel.TENANT.name()} or ${ConfigLevel.TENANT_USER.name()} level, which value is separated by commas. " +
+    s"The parameter name specifies the user names of ${ConfigLevel.TENANT_USER.name()} level, which value is separated by commas."
+
+  override def handle(service: HttpService, parameters: Map[String, String]): String =
+    service.getDynamicConfig(
+      parameters.getOrElse("LEVEL", ConfigLevel.values().mkString(","))
+        .toUpperCase(Locale.ROOT).trim,
+      parameters.getOrElse("TENANT", "").trim,
+      parameters.getOrElse("NAME", "").trim)
```

Comment:
I think
1. If set level with system level, we only show system level config
2. If set level with tenant level, without tenant id, should list all tenant level config
3. If set level with tenant level with tenant id, should only show this tenant id 's config
4. If set level with tenant user level, without setting id and name, should show all user level
5. If set level with tenant user level, with setting tenant id, should this tenant id's all user level config
6. If set level with tenant user level, with setting id and name, should show only this user's user level config
7. If not set level, but set tenant id or name, should follow above each level's behavior.